circle.yml

```diff
@@ -0,0 +1,8 @@
+dependencies:
+    pre:
+        - sudo apt-get install arduino arduino-core arduino-mk
+        - git clone https://github.com/scottdarch/Arturo && cd Arturo && sudo make install
```

Comment:
Should we move this to a submodule? Alternatively, is it available in any repos to avoid the build step?